section/authentication.html

```diff
@@ -0,0 +1,302 @@
+
+  <p>The validity of signatures is established because the user, during setup or an authorization procedure,
+    registers the base64-encoded public JWK with their wallet provider. The wallet provider then uses
+    that public key to verify subsequent signed requests (for example, grant creation, quote requests,
+    and outgoing payment creation).</p>
+
+  <p class="note" title="Privacy">
+    Reusing a single long-lived signing key across
+    multiple origins or wallet providers can enable cross-site tracking.
+    Future revisions may define per-session or per-origin derived authentication
+    keys to further reduce tracking surfaces. See the <a href="https://docs.google.com/document/d/1s9aXcLoBdLe9y_8JispFK7NVERxUhxn0orqgzZBrcBo/edit?tab=t.0#heading=h.z0cxd4l694ag">
+    directed identity proposal</a> for background.
+  </p>
+
+  <p>User agents MUST use an [=Ed25519 key pair=] to sign HTTP requests that
+    require proof-of-possession as described in this specification.</p>
+
+  <p>A generated key pair is persisted in {{Storage}}.</p>
+
+  <p class="note">The user agent SHOULD display the base64-encoded public JWK on its [=setup page=] to help
+      the user register it with their wallet provider.</p>
+
+  <p class="note">
+    Regenerating the key pair invalidates the association the user previously
+    made between their public key and their wallet. The user would have to
+    re-upload or re-register the new public key and repeat any interactive
+    authorization flows.
+  </p>
+
+  <section>
+    <h3 id="ed25519-key-pair">Ed25519 key pair</h3>
+    <p>The <dfn data-lt="key id|kid">Ed25519 key id</dfn> (<code>kid</code>) is a globally unique opaque
+      identifier bound 1:1 to an [=Ed25519 key pair=].</p>
+
+    <p>An <dfn data-lt="Ed25519 key pair|auth key pair|authentication key pair|key pair">Ed25519 key pair</dfn> is
+      the ordered pair of a [=private key=] (seed) and its corresponding [=public key=] as defined
+      in [[RFC8032]]. It is identified by a [=key id=].</p>
+    <p>
+      The <dfn data-lt="private key|Ed25519 private key|auth private key">Ed25519
+      private key</dfn> is the 32‑octet seed. It is
+      conceptually an opaque 32‑element octet array (e.g.
+      <code>std::array&lt;uint8_t, 32&gt;</code>) and SHOULD be stored and persisted
+      only in that seed form. Any expanded form needed for signing MAY be
+      derived transiently in memory and SHOULD be zeroized after use. The seed MUST
+      remain local to the user agent and MUST NOT be exposed to web content or
+      transmitted.
+    </p>
+    <p>
+      The <dfn data-lt="public key|Ed25519 public key|auth public key">Ed25519
+      public key</dfn> is the 32-octet encoded public group element. In other words,
+      it is the canonical public key bytes produced from the corresponding [=private key=].
+      The public key is intended to be shared with the user's wallet provider so the wallet
```

Comment:
Can also mention that it's viewable on the https://openpayments.dev/apis/wallet-address-server/operations/get-wallet-address-keys/ endpoint.

Reply:
good point, added as a note